README.md

```diff
@@ -1,2 +1,8 @@
 # [Sample Repo] PR Practice
 A sample repo for practicing how to create Pull Requests
+
+# Introduction
+Creating Pull Requsts (PRs) is needed when using RCS in a multi-person projects. This repo can be used to practice creating PRs.
+
+#Conclusion
```

Comment:
This conclusion can be more meaningful. It is self-evident that the PR repository is useful.